### PR TITLE
chore: fix tests to follow latest Svelte 5

### PR DESCRIPTION
**Pull request by baseballyama**
I just added Node 22 but CI does not pass in https://github.com/sveltejs/eslint-plugin-svelte/pull/750

https://github.com/sveltejs/svelte/pull/11327 changes warning messages, so I mainly copied exist tests and optimized for Svelte 5.

### CODE REVIEW

**Comment by baseballyama** [comment]
So, I think https://github.com/sveltejs/svelte/pull/11327 changes messages, so messages between Svelte 4 and Svelte 5 is not same anymore.
We need to add a feature that should be possible to declare a test specifically for Svelte 4 or Svelte 5.

**Comment by baseballyama** [comment]
We need to wait to solve this issue.

https://github.com/sveltejs/svelte/issues/11482

File: eslint.config.mjs [hunk 1]
```diff
@@ -128,7 +128,8 @@ const config = [
 		},
 		rules: {
 			'n/file-extension-in-import': 'off',
-			'n/no-unsupported-features/es-syntax': 'off'
+			'n/no-unsupported-features/es-syntax': 'off',
+			'n/no-unsupported-features/es-builtins': 'off'
```
Comment (baseballyama) [inline eslint.config.mjs:132]: I needed to disable `n/no-unsupported-features/es-builtins` for `window.btoa`.

File: docs-svelte-kit/src/lib/components/ESLintPlayground.svelte [hunk 2]
```diff
@@ -190,10 +190,9 @@
 				<ol>
 					{#each messages as msg, i (`${msg.line}:${msg.column}:${msg.ruleId}@${i}`)}
 						<li class="message">
-							<!-- svelte-ignore a11y-invalid-attribute -->
-							<a href="#" on:click={(evt) => onClickMessage(evt, msg)} class="message-link"
-								>[{msg.line}:{msg.column}]</a
-							>:
+							<button on:click={(evt) => onClickMessage(evt, msg)} class="message-link">
```
Comment (baseballyama) [inline docs-svelte-kit/src/lib/components/ESLintPlayground.svelte:193]: I resolved the warning.

**Review by ota-meshi — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**